templates/deposit-compliance/cards/cross-repository-check-card.jsx

```diff
@@ -4,29 +4,44 @@ import { Card } from 'design'
 import * as texts from 'texts/depositing'
 import { formatNumber } from 'utils/helpers'
 import ExportButton from 'components/export-button'
+import Markdown from 'components/markdown'
+
+const Content = ({ nonCompliantCount, differentCount, exportUrl }) => {
+  const template =
+    texts.crossRepositoryCheck[differentCount ? 'success' : 'failure']
+  const text = template.render({
+    nonCompliantCount: formatNumber(nonCompliantCount),
+    recordsInAnotherRepository: formatNumber(differentCount),
+  })
+
+  return (
+    <>
+      <Markdown>{text}</Markdown>
+      <ExportButton href={exportUrl}>
+        {texts.crossRepositoryCheck.download}
+      </ExportButton>
+    </>
+  )
+}
 
 const CrossRepositoryCheckCard = ({
   crossDepositLag,
-  crossDepositLagCsvUrL,
+  crossDepositLagCsvUrL: crossDepositLagCsvUrl,
```

Comment:
@Joozty it seems you made a mistake typing fast. I made a quick fix renaming it in a single file. It's not a big deal but feel free to fix it if this looks annoying to you 🙂

Reply:
yeah, thanks for noticing. Will rename it.

Reply:
actually do you mind doing it here? It's should be so hard.

Reply:
I created #341 but I would prefer to merge this one and then check #341.